config/routes.rb

```diff
@@ -4,7 +4,10 @@
   resources :products
   root 'home#index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ('/')
-  # root 'articles#index'
+  namespace :api do
+    namespace :v1 do
+      # resources :products, only: [:index]
```

Comment:
this commented syntax is better one as it's "RESTful" so please change code to use it :)